Respawn/MySqlAdapter.cs

```diff
@@ -100,7 +100,16 @@ public string BuildDeleteCommandText(GraphBuilder graph)
             return builder.ToString();
         }
 
-        public string BuildReseedSql(IEnumerable<Table> tablesToDelete) => throw new System.NotImplementedException();
+        public string BuildReseedSql(IEnumerable<Table> tablesToDelete)
+        {
+            var builder = new StringBuilder();
+            foreach (var table in tablesToDelete)
+            {
+                builder.AppendLine($"ALTER TABLE {table.GetFullName(QuoteCharacter)} AUTO_INCREMENT = 1;");
```

Comment:
Defaults to `1` for reseed value as I couldn't determine a way to detect original seed value like MS SQL